homeassistant/components/tradfri/__init__.py

```diff
@@ -142,10 +142,14 @@ async def async_keep_alive(now: datetime) -> None:
 
         gw_status = True
         try:
-            await api(gateway.get_gateway_info())
+            await api(gateway.get_gateway_info(), timeout=TIMEOUT_API)
         except RequestError:
             _LOGGER.error("Keep-alive failed")
             gw_status = False
+        except PytradfriError as exc:
+            msg = f"keep-alive {exc}"
```

Comment:
Please don't pre-format the logging message. Use the logger formatting arguments.

Reply:
Done